Update requirements, pin modules to the latest version, …

andreoliw opened the pull request:
…plus two small fixes

sloria commented on .gitignore:
```diff
@@ -45,3 +45,6 @@ output/
 bower_components
+
+# PyCharm
+.idea/
```
These OS- and editor-generated files should go in a global gitignore: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore . Project gitignores should contain only language- and project- specific files.

sloria commented:
Thanks for the PR! Just one suggestion to remove the editor-generated files from the .gitignores. Other than that, this is good to merge.

andreoliw commented:
Thanks for the answer.

You're right, I removed those files from `.gitignore`.
It's ready for review again.

---

By the way, I personally use `flake8` plus some plugins to check code style.
Would you accept a PR if I added a `manage.py flake8` (or `manage.py lint`) command to the cookiecutter template?
It would be a bigger PR, I guess, not as atomic as this one.

If you don't want that, then I'll just do it in my fork, and it's fine.

sloria commented:
Thanks for updating.

I would certainly review a PR adding linting. It's been something I've been meaning to add.
